hera/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:14.04
+MAINTAINER John Vivian, jtvivian@gmail.com
+
+# install dependencies first
+RUN apt-get update  && apt-get install -y \
+		build-essential \
+		liblzma-dev \
+		libbz2-dev \
+		libz-dev \
+		git \
+		wget
+
+
+# Install cmake 3.9.1
+WORKDIR /
+# git clone hera since we're here
+RUN git clone https://github.com/bioturing/hera.git
+# On with cmake
+RUN wget https://cmake.org/files/v3.9/cmake-3.9.1.tar.gz
+RUN tar -xzvf cmake-3.9.1.tar.gz
+WORKDIR /cmake-3.9.1
+RUN ./bootstrap
+RUN make -j4
+RUN make install
+
+# install hera from source
+WORKDIR /hera
+RUN git checkout 53dd2a1927d86d2299f16cf265d3f0e259c0e3df
+RUN chmod +x build.sh
+RUN ./build.sh
+
+# Boilerplate
+RUN mkdir /data
```

Comment:
I don't think you need to make `/data`, when you `-v` mount, it should be created. I would remove this.

Reply:
I thought the convention was that the WORKDIR is always set by default to /data

Reply:
I don't think we've typically done that.

Reply:
Oops. You are correct. Disregard me!